tests/test_jsdcov.py

```diff
@@ -13,15 +13,16 @@
 
 from activedata_etl.transforms import cov_to_es
 from activedata_etl.transforms.jscov_to_es import process_jscov_artifact
+from mo_dots import Null, Data
 from pyLibrary.aws.s3 import PublicBucket
 from test_gcov import Destination
 
 
 class TestJsdov(unittest.TestCase):
     def test_one_url(self):
-        key="tc.1051816"
-        url="https://queue.taskcluster.net/v1/task/PNzAZrN7SUeKMK0_-wZr0Q/runs/0/artifacts/public/test_info//code-coverage-gcda.zip"
-        destination = Destination("results/ccov/lcov_parsing_result.json.gz")
+        key="tc.472127"
```

Comment:
notice this `key` is only ever used for debugging.  You can set it to `Null` if you want.